src/app/suggestion-notifications/qa/events/quality-assurance-events.component.html

```diff
@@ -0,0 +1,218 @@
+              </tr>
+              </thead>
+              <tbody>
+                <tr *ngFor="let eventElement of (eventsUpdated$ | async); let i = index">
+                  <td>{{eventElement?.event?.trust}}
+                  </td>
+                  <td><a *ngIf="eventElement?.target"
+                         target="_blank"
+                         [routerLink]="['/items', eventElement?.target?.id]">{{eventElement.title}}</a>
+                    <span *ngIf="!eventElement?.target">{{eventElement.title}}</span>
+                  </td>
+                  <td *ngIf="showTopic.indexOf('/PID') !== -1">
+                    <p><span class="small">{{'quality-assurance.event.table.pidtype' | translate}}</span>&nbsp;<span class="badge badge-info">{{eventElement.event.message.type}}</span></p>
+                    <p><span class="small">{{'quality-assurance.event.table.pidvalue' | translate}}</span><br>
+                      <a *ngIf="hasPIDHref(eventElement.event.message); else noPID" href="{{getPIDHref(eventElement.event.message)}}" target="_blank">
+                        {{eventElement.event.message.value}}
+                      </a>
+                      <ng-template #noPID><span class="badge badge-info">{{eventElement.event.message.value}}</span></ng-template>
+                    </p>
+                  </td>
+                  <td *ngIf="showTopic.indexOf('/SUBJECT') !== -1">
+                    <p><span class="small">{{'quality-assurance.event.table.subjectValue' | translate}}</span><br><span class="badge badge-info">{{eventElement.event.message.value}}</span></p>
+                  </td>
+                  <td *ngIf="showTopic.indexOf('/ABSTRACT') !== -1">
+                    <p class="abstract-container" [class.show]="showMore">
+                      <span class="small">{{'quality-assurance.event.table.abstract' | translate}}</span><br>
+                      <span class="text-ellipsis">{{eventElement.event.message.abstract}}</span>
+                    </p>
+                    <button class="btn btn-outline-primary btn-sm" (click)="showMore = !showMore">
+                      <i *ngIf="!showMore" class="fas fa-angle-down"></i>
+                      <i *ngIf="showMore" class="fas fa-angle-up"></i>
+                      {{ (showMore ? 'quality-assurance.event.table.less': 'quality-assurance.event.table.more') | translate }}
+                    </button>
+                  </td>
+                  <td *ngIf="showTopic.indexOf('/PROJECT') !== -1">
+                    <p>
+                      {{'quality-assurance.event.table.suggestedProject' | translate}}
+                    </p>
+                    <p>
+                      <span class="small">{{'quality-assurance.event.table.project' | translate}}</span><br>
+                      <a href="https://explore.openaire.eu/search/project?projectId={{ eventElement.event.message.openaireId}}" target="_blank">{{eventElement.event.message.title}}</a>
```

Comment:
This link has a hardcoded URL back to OpenAIRE.  We may want to make this configurable at some point in case the URL changes